app/assets/stylesheets/application.css

```diff
@@ -0,0 +1,66 @@
+/*
+ * This is a manifest file that'll be compiled into application.css, which will include all the files
+ * listed below.
+ *
+ * Any CSS (and SCSS, if configured) file within this directory, lib/assets/stylesheets, or any plugin's
+ * vendor/assets/stylesheets directory can be referenced here using a relative path.
+ *
+ * You're free to add application-wide styles to this file and they'll appear at the bottom of the
+ * compiled file so the styles you add here take precedence over styles defined in any other CSS
+ * files in this directory. Styles in this file should be added after the last require_* statement.
+ * It is generally better to create a new file per style scope.
+ *
+ *= require_tree .
+ *= require_self
+ */
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  background-color: #ebebeb;
+}
+
+.back-primary {
+  background-color: #3778c2 !important;
+}
+
+.back-secondary {
+  background-color: #434b54;
+}
+
+.back-green {
+  background-color: #5fb523;
+}
+
+.c_card img {
+  width: 70px;
+  height: 70px;
+}
+
+.card_info {
+  flex: 1;
+  display: flex;
+  justify-content: space-between;
+  height: 70px;
+}
+
+.card_info a {
+  text-decoration: none;
+  font-size: 24px;
+  font-weight: 600;
+  color: #434b54;
+}
+
+.card_info span {
+  font-size: 12px;
+  align-self: flex-end;
+}
+
+.footer {
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  right: 0;
+}
```

Comment:
- [ ] The official font face for this application based on the design given is **_Proxima Nova Bold_** for headers and **_Proxima Nova Light_** for the general text of the application. Kindly apply these design rules as adherence to design rules makes you a better developer. 👍 😎  

![image](https://github.com/yegonkimutai/budget-app/assets/77106962/b261f95a-51c3-4348-b2aa-c399585dd108)

Reply:
Thank you @Gambit142 for all your comments 🙏